features.html

```diff
@@ -3,25 +3,24 @@
 ---
 <h2>Features</h2>
 <div class="video">
-  <iframe src="//player.vimeo.com/video/1239618" width="500" height="333" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
-  <a href="http://www.vimeo.com/1239618?pg=embed&amp;sec=1239618">VuFind®: Social Features</a> from
-  <a href="http://www.vimeo.com/user433393?pg=embed&amp;sec=1239618">Falvey Memorial Library</a> on
-  <a href="http://vimeo.com?pg=embed&amp;sec=1239618">Vimeo</a>.
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/6Nf6sDSlQqg?si=p00c27j_OD-z5BL8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
Replacing 2008-era video with more recent webinar. I kept a link to the old video in the wiki on the videos page, under "Historical."